### PR TITLE
Keep node versions in sync

### PR DESCRIPTION
**Pull request by NotMyFault**
The change proposed switches to renovate to ensure the node version used to build on the CI and used in the Dockerfile remains aligned, see https://github.com/jenkins-infra/incrementals-publisher/pull/73 diverging between the node versions.

 Additionally, I removed the unused .nvmrc file, given 14 can't be used anyway.

### CODE REVIEW

**Comment by lemeurherve** [comment]
Maybe replace dependabot all together by renovabot at the same time?

**Comment by NotMyFault** [comment]
> Maybe replace dependabot all together by renovabot at the same time?

For sure!

**Review by lemeurherve — APPROVED** [review]

File: .tool-versions
Comment (lemeurherve) [inline .tool-versions:1]: We need to keep this file for asdf.

**Review by lemeurherve — CHANGES_REQUESTED** [review]

File: .tool-versions
Comment (NotMyFault) [reply .tool-versions:1]: Oic

**Review by lemeurherve — APPROVED** [review]

File: .tool-versions [hunk 1]
```diff
@@ -1 +1 @@
-nodejs 18.16.1
+nodejs 21.6.1
```
Comment (timja) [inline .tool-versions:1]: Use an old version to check renovate works maybe?

Comment (NotMyFault) [reply .tool-versions:1]: https://github.com/NotMyFault/incrementals-publisher/issues/6

Applied this PR as patch https://github.com/NotMyFault/incrementals-publisher/commit/f9a1e7bff75117ce74bd22d2155db141c3f9c5e5 and renovate detects it in its dashboard

Comment (NotMyFault) [reply .tool-versions:1]: Actually, I removed the regex check after renovate picked it up as intended in the issue linked above.

**Review by timja — APPROVED** [review]

File: .tool-versions (hunk 1 above)
Comment (dduportal) [reply .tool-versions:1]: Is this possible to keep an LTS line to avoid bad surprises?

**Comment by NotMyFault** [comment]
I've requested an installation of renovate on this repo.

**Comment by lemeurherve** [comment]
Repo added:

image

**Pull request merged**